lib/gotta/mod/cli/uninstall.rb

```diff
@@ -19,11 +19,10 @@ module Gotta
   module Mod
     module CLI
       class UninstallMod < Gotta::CLI::Command::Base
-        require 'gotta/project'
-
         command 'uninstall mod :mod_name'
 
         def run!
+          require 'gotta/project'
```

Comment:
Should this just be `require 'gotta/project/config'`?

Reply:
@parruda I think one of 
```suggestion
          require 'gotta/project/config'
```